packages/core/src/Account.test.ts

```diff
@@ -88,6 +91,12 @@ describe('Account', () => {
     user: 'aaf9a833-ef30-4c22-86a0-9adc8a15b3b4',
   };
 
+  beforeEach(() => {
+    // @ts-ignore
+    global.crypto = nodeCrypto.webcrypto;
+    global.window = {crypto: nodeCrypto.webcrypto, btoa: i => i};
```

Comment:
There might be a better way to do this 🤔

Reply:
That should be done in a setup file. Same for the indexdb polyfill